### DISCUSSION TITLE
Change project-wide execution options for Mermaid

### DISCUSSION THREAD

**Discussion by dhicks** [quarto-dev/quarto-cli#1484] [Q&A]
I'm working on a project that includes, in effect, an extensive Mermaid tutorial.  I'd like to set all of the Mermaid blocks to `echo: true`.  Either setting `echo: true` in `_quarto.yml` doesn't seem to do the trick or I'm confused about where to put it. Non-working example attached. 

[_quarto.yml.zip](https://github.com/quarto-dev/quarto-cli/files/9162064/_quarto.yml.zip)

**Comment by cscheid** [comment]
This should work, it's most likely a plain bug.

**Reply by cscheid** [reply]
This should be fixed as of https://github.com/quarto-dev/quarto-cli/commit/7a9558d74deec0d06b39ea9d58f89f0fe56678ff.
